compute/server-groups.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Server groups
+=============
+
+By creating groups of instances, its possible to assign various functions from the platform onto the members of the group. The most common feature is "affinity" (or "anti affinity") wherein you can force instances to be on the same underlying hardware ("hypervisor") - or vice versa (to force instances onto separate hypervisors). 
+
+Affinity types
```

Comment:
The CLI says they are called ``Policies`` so ``Affinity types`` here feels inconsistent